Dev cam arm

roxyvallee opened the pull request:
**What :** 
Hand and flute interaction added to the camera. If you play a note, the flute will appear in front of the player.
If you don't play a note in a certain amount of time, the flute will disappear. 

**How to test :**
Press Y, U, I, O, P keys
See if the flute is changing positions depending of the key pressed
Wait a moment ( 3 secondes ) to see if the flute disappears

Commit:
[Add]: Hand and flute object in the scene

add Hand and Flute object, create a script for the interaction between the keyboard and the objects

Commit:
[Add]: Timer on playing note

if the player is out of the timer, the flute disappears.

Commit:
[Fix]: delete shadows for object scene

Commit:
Merge remote-tracking branch 'origin/master' into dev_cam_arm

MonicaLis reviewed (approved):
Works very well on my side, well done :-)

piptouque commented:
The flute and hand disappear after a set time even though the note is still playing.

piptouque commented on LullabyProject/Assets/Scripts/HandMovement.cs:
```diff
@@ -0,0 +1,67 @@
+    // Start is called before the first frame update
+    void Start()
+    {
+        
```
(It's good practice to check that `keys.Length == positionNote.Length`). Or even `== 5`, since we hard-code it.

roxyvallee replied:
Yes I will do that !

piptouque reviewed (changes requested):
The flute and hand disappear after a set time even though the note is still playing.

Commit:
[Fix]: Corrected HandMovement script

roxyvallee commented:
@piptouque Indeed but we need to associate our script FlutePlayer.cs and HandMovement.cs to see if I can check in my code if the note is still playing.

piptouque commented:
@roxyvallee Got it, it makes sense (I have an idea how to do this). Well done !

Lchikar reviewed (approved):
link between HandMovement.cs and FlutePlayer.cs should be the next priority, however it is eventually not necessary for the prototype stage

Commit:
[Improvement]: Added Command queue and State machine to FlutePlayer.

Also refactored HandMovement to reflect the changes.
Should allow for handling animations.

Commit:
Merge branch 'master' into dev_cam_arm
